filebeat/docs/getting-started.asciidoc

```diff
@@ -56,48 +56,14 @@ include::{libbeat-dir}/tab-widgets/install-widget.asciidoc[]
 include::{libbeat-dir}/shared/connecting-to-es.asciidoc[]
 
 [float]
-[[enable-modules]]
-=== Step 3: Enable and configure data collection modules
+[[collect-data]]
+=== Step 3: Collect data
```

Comment:
I like the change here. I think you could say "Collect log data" to be more specific.